go/serialization.go

```diff
@@ -151,6 +151,33 @@ func ReadTimestampedEntryInto(r io.Reader, t *TimestampedEntry) error {
 	return nil
 }
 
+// SerializeTimestampedEntry writes timestamped entry to Writer.
```

Comment:
Could you add a comment to mention that in case of error, `w` might have garbage on it?